bin/all_sky_search/pycbc_fit_sngls_over_multiparam

```diff
@@ -88,7 +88,11 @@ def smooth_tophat(nabove, invalphan, ntotal, dists):
                             idx_within_area)
 
 
-def smooth_n_closest(nabove, invalphan, ntotal, dists, total_trigs=500):
+_default_total_trigs = 500
```

Comment:
add a comment here to explain what this value will be used for (the name is slightly cryptic)